hoverxref/_static/js/hoverxref.js_t

```diff
@@ -144,6 +138,33 @@ $(document).ready(function() {
             );
         }
     })
+}
+
+
+$(document).ready(function() {
+    // Remove ``title=`` attribute for intersphinx nodes that have hoverxref enabled.
+    // It doesn't make sense the browser shows the default tooltip (browser's built-in)
+    // and immediately after that our tooltip was shown.
+
+    // Support lazy-loading here by switching between on-page-load (calling .tooltipster directly)
+    // or delaying it until after a mouseenter or click event.
+    // On large pages (the use case for lazy-loaded tooltipster) this moves dom manipulation to
+    // on-interaction, which is known to be less performant on small pages (as per tooltipster docs),
+    // but on massive docs pages fixes an otherwise severe page load stall when tooltipster
+    // manipulates the html for every single tooltip at once.
+    {% if hoverxref_tooltip_lazy %}
+    $('.{{ hoverxref_css_class_prefix }}hoverxref.{{ hoverxref_css_class_prefix }}tooltip').one('mouseenter click touchstart tap', function(event) {
+        $(this).removeAttr('title');
+        {# If we're lazy-tipstering, we've used up the click event, so open the modal too with tooltipster('open') #}
+        addTooltip($(this)).tooltipster('open');
+    });
+    {% else %}
+    $('.{{ hoverxref_css_class_prefix }}hoverxref.external').each(function () { $(this).removeAttr('title') });
```

Comment:
Shouldn't we move this outside the if/else and put to below the comment in line 145? This code is required for both paths,.

Reply:
I think the intention is to keep the `title` attr until tooltipster is actually triggered, which happens "lazily".

Reply:
That is the intention. I could add a comment to label it for that? Editing the dom on page load to the degree found in the disnake docs (150kish titles) is a significant amount of effort that rarely if ever needs to be done in the first place (since a user usually doesn't hover over more than a few hoverxref links, anyway, and certainly not in the thousands) so deffering it with popup generation makes sense to me.

I don't currently have the measurements, but it was somewhere in the ballpark of 0.5-1.5s of delay from that line alone, which is on one hand reasonable, on the other the entire point of this feature is to remove that behavior in the first place.